app/components/qualification_title_component.rb

```diff
@@ -5,7 +5,12 @@ def initialize(qualification:)
 
   def qualification_type
     if @qualification.level == 'gcse' && @qualification.missing_qualification?
-      'GCSE'
+      I18n.t('application_form.gcse.qualification_types_abbreviated.gcse')
+    elsif @qualification.level == 'gcse'
+      I18n.t(
+        "application_form.gcse.qualification_types_abbreviated.#{@qualification.qualification_type}",
```

Comment:
Alternatively, we could set these as i18n strings for ActiveRecord:

```
  activerecord:
    attributes:
      application_qualification/qualification_type:
        gcse: GCSE
        gce_o_level: GCE O Level
        scottish_national_5: Scottish National 5
        other_uk: Other UK
        missing: I don’t have this qualification yet
```

And call like this?

```
      ApplicationQualification.human_attribute_name(
        "qualification_type.#{@qualification.qualification_type}", default: @qualification.qualification_type
      )
```

Reply:
thanks, that looks a lot better, done

Reply:
thanks, that looks a lot better, done